docs/libcurl/opts/CURLOPT_KRB5_CLIENT_KTNAME.3

```diff
@@ -0,0 +1,53 @@
+.\" **************************************************************************
+.\" *                                  _   _ ____  _
+.\" *  Project                     ___| | | |  _ \| |
+.\" *                             / __| | | | |_) | |
+.\" *                            | (__| |_| |  _ <| |___
+.\" *                             \___|\___/|_| \_\_____|
+.\" *
+.\" * Copyright (C) 2016, Daniel Stenberg, <daniel@haxx.se>, et al.
+.\" *
+.\" * This software is licensed as described in the file COPYING, which
+.\" * you should have received as part of this distribution. The terms
+.\" * are also available at http://curl.haxx.se/docs/copyright.html.
+.\" *
+.\" * You may opt to use, copy, modify, merge, publish, distribute and/or sell
+.\" * copies of the Software, and permit persons to whom the Software is
+.\" * furnished to do so, under the terms of the COPYING file.
+.\" *
+.\" * This software is distributed on an "AS IS" basis, WITHOUT WARRANTY OF ANY
+.\" * KIND, either express or implied.
+.\" *
+.\" **************************************************************************
+.\"
+.TH CURLOPT_KRB5_CLIENT_KTNAME 3 "26 March 2016" "libcurl 7.49.0" "curl_easy_setopt options"
+.SH NAME
+CURLOPT_KRB5_CLIENT_KTNAME \- set Kerberos client keytab to use in GSSAPI
+.SH SYNOPSIS
+#include <curl/curl.h>
+
+CURLcode curl_easy_setopt(CURL *handle, CURLOPT_KRB5_CLIENT_KTNAME, char *keytab);
+.SH DESCRIPTION
+Pass a pointer to a NULL terminated string as parameter. The string represents
+the KRB5 client keytab to use instead of the default client keytab (file path
+or other types, similar to KRB5_CLIENT_KTNAME environment variable).
+
+.SH DEFAULT
+NULL
+.SH PROTOCOLS
+All protocols using KRB5 mechanism in GSSAPI: HTTP, FTP, etc.
```

Comment:
For consistency with other areas of cURL / libcurl documentation "KRB5 mechanism" => "Kerberos V5 mechanism"